Dockerfile

```diff
@@ -25,5 +25,4 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone &
 WORKDIR /oot
 RUN git config --global --add safe.directory /oot
 
-CMD ["/bin/sh", "-c", \
-    "echo 'usage:\n  docker run --rm --mount type=bind,source=\"$(pwd)\",destination=/oot oot make -j$(nproc) setup\n  docker run --rm --mount type=bind,source=\"$(pwd)\",destination=/oot oot make -j$(nproc)'"]
+CMD make -j $(nproc) setup && make -j $(nproc) && /bin/bash
```

Comment:
am I understanding right that docker users will have to run `setup` every time they want to build?
I think this was being talked about in a previous thread but i still dont get it

Reply:
If one continues to use the same image you can get around that. I'm not too sure how often you would need to rerun make setup, but they could treat the running container the same as a Linux box until a restart.

Reply:
I guess the concern is, if you shut down the container and restart it later, maybe you don't want to rebuild everything when you restart (since it has already been built)?

Reply:
Yes, as it stands now, if the user shuts down this image, they will have to rebuild. This is the same as the behavior that currently stands *I think*, as the --rm flag removes the image the second it shuts down. I believe that I can address this by copying into the image itself, and maybe do the build , instead of against the user's filesystem. Not 100% sure best path.

Reply:
if you want to explore this in the future feel free to. I think you are the first person even touching this in years, so tbh you do what you find best.

If this is the same as current behavior then lgtm